exp/api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -93,10 +93,14 @@ type AzureManagedControlPlaneSpec struct {
 	// +optional
 	IdentityRef *corev1.ObjectReference `json:"identityRef,omitempty"`
 
-	// AadProfile is Azure Active Directory configuration to integrate with AKS for aad authentication.
+	// AADProfile is Azure Active Directory configuration to integrate with AKS for aad authentication.
```

Comment:
I know this is nitpicky but let's not make his comment change here? (I assume your IDE did it for you :))

Reply:
Ok, removed.